docs/architecture/modernize-desktop/whats-new-dotnet-core-desktop.md

```diff
@@ -0,0 +1,149 @@
+
+The .NET Standard enables the following key scenarios:
+
+- Defines uniform set of base class libraries APIs for all .NET implementations to implement, independent of the workload.
+- Enables developers to produce portable libraries that are usable across .NET implementations, using this same set of APIs.
+- Reduces or even eliminates conditional compilation of shared source due to .NET APIs, only for OS APIs.
+
+.NET Standard is the evolution of PCLs and here are the fundamental differences between .NET Standard and PCLs:
+
+- .NET Standard is a set of curated APIs, picked by Microsoft, PCLs are not.
+- The APIs that a PCL contains are dependent on the platforms that you choose to target when you create a PCL. This makes a PCL only sharable for the specific targets that you choose.
+- .NET Standard is platform-agnostic, it can run anywhere, on Windows, macOS, Linux, and so on.
+- PCLs can also run cross-platform, but they have a more limited reach. PCLs can only target a limited set of platforms.
+
+## .NET Core 3.0 new Desktop features
+
+### Support for Windows Forms and WPF
+
+Windows Forms and WPF are part of .NET Core 3.0. Both presentation frameworks are only for the Windows OS and they are not cross-platform. You can think of WPF as a rich layer over DirectX and Windows Forms as a thinner layer over GDI+. WPF and Windows Forms do a great job of exposing and exercising much of the desktop application functionality in Windows. With .NET Core 3.0. Windows Forms and WPF become available for .NET Core as well as for .NET Framework. Now you can start your new desktop applications targeting .NET Core and migrate your existing ones from .NET Framework to .NET Core.
+
+A new version of .NET Standard, version 2.1, was released at the same time. As expected, all new .NET Standard APIs are part of .NET Core 3.0.
+
+You can build desktop applications with C#, F#, and VB with .NET Core 3.0.
+
+Also, important to notice that both Windows Forms and WPF implementations for .NET Core 3 are being open-sourced.
+
+### XAML Islands
+
+XAML Islands is a set of components for developers to use the new Windows 10 controls (UWP XAML controls) in their current WPF, Windows Forms, and native Win32 apps (like MFC). You can have your "islands" of UWP XAML controls wherever you want inside your Win32 apps.
+
+These XAML Islands are possible because Windows 10 1903 update introduces a set of APIs that allows hosting UWP XAML content in Win32 windows using windows handlers (HWnds). Notice that only apps running on Windows 10 1903 and above can use XAML Islands.
```

Comment:
```suggestion
These XAML Islands are possible because Windows 10, version 1903 introduces a set of APIs that allows hosting UWP XAML content in Win32 windows using windows handlers (HWnds). Notice that only apps running on Windows 10 1903 and above can use XAML Islands.
```